packages/rocketchat-authorization/client/views/permissions.html

```diff
@@ -1,36 +1,76 @@
+<template name="permissionsTable">
```

Comment:
Created a template for the table as both permission tables follow the same scheme.